homeassistant/bootstrap.py

```diff
@@ -323,13 +325,34 @@ async def _async_set_up_integrations(
 ) -> None:
     """Set up all the integrations."""
 
+    hass.data[DATA_SETUP_STARTED] = {}
+
     async def async_setup_multi_components(domains: Set[str]) -> None:
         """Set up multiple domains. Log on failure."""
+
+        async def _async_log_pending_setups() -> None:
+            """Periodic log of setups that are pending for longer than LOG_SLOW_STARTUP_INTERVAL."""
+            while True:
+                await asyncio.sleep(LOG_SLOW_STARTUP_INTERVAL)
+                remaining = [
+                    domain
+                    for domain in domains
+                    if domain not in hass.config.components
```

Comment:
Do we need this check ?

Reply:
No, will remove.  I need to fix the race condition in the test as well.

Reply:
Done